modules/spring/src/main/java/org/apache/ignite/transactions/spring/AbstractSpringTransactionManager.java

```diff
@@ -0,0 +1,303 @@
+        IgniteTransactionObject txObj = (IgniteTransactionObject)status.getTransaction();
+        TransactionProxy tx = txObj.getTransactionHolder().getTransaction();
+
+        if (status.isDebug() && log.isDebugEnabled())
+            log.debug("Committing Ignite transaction: " + tx);
+
+        try {
+            tx.commit();
+        }
+        catch (Exception e) {
+            throw new TransactionSystemException("Could not commit Ignite transaction", e);
+        }
+    }
+
+    /** {@inheritDoc} */
+    @Override protected void doRollback(DefaultTransactionStatus status) throws TransactionException {
+        IgniteTransactionObject txObj = (IgniteTransactionObject)status.getTransaction();
+        TransactionProxy tx = txObj.getTransactionHolder().getTransaction();
+
+        if (status.isDebug() && log.isDebugEnabled())
+            log.debug("Rolling back Ignite transaction: " + tx);
+
+        try {
+            tx.rollback();
+        }
+        catch (Exception e) {
+            throw new TransactionSystemException("Could not rollback Ignite transaction", e);
+        }
+    }
+
+    /** {@inheritDoc} */
+    @Override protected void doSetRollbackOnly(DefaultTransactionStatus status) throws TransactionException {
+        IgniteTransactionObject txObj = (IgniteTransactionObject)status.getTransaction();
+        TransactionProxy tx = txObj.getTransactionHolder().getTransaction();
+
+        assert tx != null;
+
+        if (status.isDebug() && log.isDebugEnabled())
+            log.debug("Setting Ignite transaction rollback-only: " + tx);
+
+        tx.setRollbackOnly();
+    }
+
+    /** {@inheritDoc} */
+    @Override protected void doCleanupAfterCompletion(Object transaction) {
+        IgniteTransactionObject txObj = (IgniteTransactionObject)transaction;
+
+        // Remove the transaction holder from the thread, if exposed.
+        if (txObj.isNewTransactionHolder()) {
+            TransactionProxy tx = txObj.getTransactionHolder().getTransaction();
+            TransactionSynchronizationManager.unbindResource(txFactory);
+
+            if (log.isDebugEnabled())
+                log.debug("Releasing Ignite transaction: " + tx);
+        }
+
+        txObj.getTransactionHolder().clear();
+    }
+
+    /** {@inheritDoc} */
+    @Override protected boolean isExistingTransaction(Object transaction) throws TransactionException {
+        IgniteTransactionObject txObj = (IgniteTransactionObject)transaction;
+
+        return (txObj.getTransactionHolder() != null && txObj.getTransactionHolder().isTransactionActive());
+    }
+
+    /** {@inheritDoc} */
+    @Override public Object getResourceFactory() {
+        return txFactory;
+    }
+
+    /**
+     * @param isolationLevel Spring isolation level.
+     * @return Ignite isolation level.
+     */
+    private TransactionIsolation convertToIgniteIsolationLevel(int isolationLevel) {
+        TransactionIsolation isolation = dfltTxIsolation;
+        switch (isolationLevel) {
```

Comment:
Add missed line breaks according to the code style:
https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines

Reply:
Done.